.pre-commit-config.yaml

```diff
@@ -1,15 +1,15 @@
 repos:
 - repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v3.3.0
+  rev: v3.4.0
   hooks:
     -   id: check-toml
     -   id: check-yaml
 - repo: https://github.com/nbQA-dev/nbQA
-  rev: 0.5.4
+  rev: 0.5.6
   hooks:
     - id: nbqa-black
       additional_dependencies: [black==20.8b1]
-      files: ^Rethinking_2/
+      files: ^(Rethinking_2|BSM)/
```

Comment:
Out of curiosity, why do you need to change this? Wasn't the previous pattern more permissive?

Reply:
`files` specifies which files to run a formatter. It takes a regular expression which is matched with `re.search`. So before, `^Rethinking_2/` would only have matched the `Rethinking_2` folder. The new one matches the `BSM` one as well, and so is less restrictive. The idea is to make the pattern less and less restrictive until it covers all files

Reply:
Ah ok, I had it backwards -- I though the previous one was recursive. All good then, thanks!